### DISCUSSION TITLE
Try To Make An Audio Editor And Cutscene If Possible!

### DISCUSSION THREAD

**Discussion by Mariogamer12** [AliveTeam/alive_reversing#1593] [Ideas]
**Audio** Is Possible Most Likely **Cutscenes!** Are Not I'm Not Sure About This!

**Comment by mouzedrift** [comment]
this is a planned feature but there is still a lot of stuff that needs to be done before that can happen

**Reply by Mariogamer12** [reply]
Oh! Cool I Didn't Know!

**Reply by Mariogamer12** [reply]
You Didn't Note That In Your Roadmap And Why?

**Reply by Mariogamer12** [reply]
Also My Dream Has Come True! I Always Wanted To Edit Cutscenes! I Love Your Team And Everyone Helping!  I Can't Wait (Sorry For The Bad My English I'm So Happy!)

**Reply by mouzedrift** [reply]
> You Didn't Note That In Your Roadmap And Why?

we can't possibly add every planned feature to the roadmap because there are just too many. i guess you could say that this counts as proper modding support (including editing cutscenes and music/sounds)
![image](https://user-images.githubusercontent.com/43358824/222409834-44db24ff-d960-4978-8fe4-1d5ddcc6cc42.png)
